build/webpack/_production.js

```diff
@@ -16,17 +16,37 @@ export default (webpackConfig) => {
   }
 
   debug('Apply ExtractTextPlugin to CSS loaders.')
+  let appliedExtractCSS = false
   webpackConfig.module.loaders = webpackConfig.module.loaders.map(loader => {
-    if (/css/.test(loader.test)) {
-      const [first, ...rest] = loader.loaders
+    if (!loader.loaders) {
+      return loader
+    }
+
+    let hasCSSLoader = false
+    loader.loaders.forEach(styleLoader => {
```

Comment:
Hey, this could probably be a bit short (while also removing the initial `loader.loaders` check at the top):

```js
if (
  !loader.loaders ||
  loader.loaders.reduce((hasLoader, loader) => {
    return hasLoader || /css/.test(loader.split('?')[0])
  }, false)
) {
  return
}
```

Thoughts?